<a href="https://colab.research.google.com/github/Bestcoder-123/Tellesales-Dashboard/blob/main/telesale.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

## TODAYS DATA ANALYSIS

$y=\beta_{0}+\alpha_{1}x$\
sdrhetritwe

In [18]:
import pandas as nakitari
import scipy
import numpy
import sys
data  = nakitari.read_csv("test.csv")
data.head(5)

Code      Branch Daily Sales Target Amount Sales (LCY)  \
0          ANN KATHAMBI  BEIBOR THK                201,000.00   86,172.00   
1           ANTA KARIMI  BEIBO KITE                230,000.00  107,659.00   
2    CATHERINE MWONGELI  BEIBORA GI                450,000.00   15,763.00   
3  CYNTHIA MUASYA MWIKA  BEIBORA HQ                544,000.00  140,918.00   
4        DAVID NYONGESA  BEI RONGAI                250,000.00   58,722.00   

  % Score  Daily Customers Target  No. of Customer Orders % Call Conversion  \
0    43%                    200.0                    38.0              19%    
1    47%                    200.0                    42.0              21%    
2     4%                    200.0                     6.0               3%    
3    26%                    200.0                    39.0              20%    
4    23%                    150.0                    27.0              18%    

  Profit (LCY)  Profit %  
0     5,892.00       6.8  
1     8,369.00       7.8  
2     1,243.00       7.9  
3     9,973.00       7.1  
4     4,874.00       8.3

|index|Code|Branch|Daily Sales Target Amount|Sales \(LCY\)|% Score|Daily Customers Target|No\. of Customer Orders|% Call Conversion|Profit \(LCY\)|Profit %|
|---|---|---|---|---|---|---|---|---|---|---|
|0|ANN KATHAMBI|BEIBOR THK|201,000\.00|86,172\.00|43% |200\.0|38\.0|19% |5,892\.00|6\.8|
|1|ANTA KARIMI|BEIBO KITE|230,000\.00|107,659\.00|47% |200\.0|42\.0|21% |8,369\.00|7\.8|
|2|CATHERINE MWONGELI|BEIBORA GI|450,000\.00|15,763\.00|4% |200\.0|6\.0|3% |1,243\.00|7\.9|
|3|CYNTHIA MUASYA MWIKA|BEIBORA HQ|544,000\.00|140,918\.00|26% |200\.0|39\.0|20% |9,973\.00|7\.1|
|4|DAVID NYONGESA|BEI RONGAI|250,000\.00|58,722\.00|23% |150\.0|27\.0|18% |4,874\.00|8\.3|

In [17]:
sys.set_int_max_str_digits(1000000000000000)
y=2**1000000
print(y)

OverflowError: Python int too large to convert to C int